.gitignore

```diff
@@ -1,6 +1,7 @@
 # Generated by Cargo
 # will have compiled files and executables
 /target/
+*/target/
```

Comment:
First, if a rust project was in a workspace, the build files are always in the top `target` directory, so, I think this line is unnecessary.

And, if you really want to add the non-top `target` directory, you can just change the `/target/` to `target/`.

Reply:
`*/target/` == `target/` ?

Reply:
`target/ == '/target/' + '/*/target/' + '/*/*/target/' +  '/*/*/*/target/' + ...`

Reply:
While running `cargo bench --features "sha3hash` under the `cita-common-benches`, cargo would create a `target` dir under the `cita-common-benches`. I did not dive into the reason.

Reply:
Oh, I see, [`criterion`] stores the benchmark results into that target.

[`criterion`]: https://crates.io/crates/criterion

Reply:
1. `target/` include `/target/`.
2. I think add `criterion/` is enough and more suitable.

Reply:
OK!